app/client/stylesheets/_home.scss

```diff
@@ -30,29 +30,59 @@
     }
     /* Hero */
     #hero {
-        background-image: url('../assets/images/hero.jpg');
+        background-image: linear-gradient(to right, #2a2a2a, #191919);
         background-size: cover;
         background-position: center center;
         height: calc(100% + 49px);
-    }
 
-    #hero::before {
-        content: '';
-        background-image: linear-gradient(to bottom right, #02ccba 0%, #3d99be 31%, #9848d5 100%);
-        opacity: 0.8;
-        width: 100%;
-        height: calc(100% + 50px);
-        position: absolute;
+        .landing-page-logo-container {
+            position:relative;
+            top:0; 
+            left:0;
+        }
+
+        .before-logo {
+            position:relative;
+            top:0; 
+            left:0; 
+            width: 100%;
+            opacity: 0;
+            animation: fadeIn 0.5s ease-in 0.5s forwards;
+        }
+
+        .after-logo {
+            position: absolute;
+            top:0; 
+            left:0;
+            width: 100%;
+            opacity: 0;
+            animation: fadeIn 0.8s ease-in 0.8s forwards;
+        }
+
+        @keyframes fadeIn {
```

Comment:
we already have this keyframe as a global utility class in `global.scss` https://github.com/hackuci/quill/blob/develop/app/client/stylesheets/_global.scss#L104